Hcal/include/Hcal/Event/VisiblesVetoResult.h

```diff
@@ -0,0 +1,85 @@
+/*
+ *@file VisiblesVetoResult.h
+ *@brief Class used to encapsulate the results obstained
+         from VisiblesVetoProcessor
+ *@author Tyler Horoho, University of Virginia
+ */
+
+#ifndef EVENT_VISIBLESVETORESULT_H_
+#define EVENT_VISIBLESVETORESULT_H_
+
+#include <iostream>
+
+//   ROOT   //
+#include <TObject.h>
+
+namespace ldmx {
+
+class VisiblesVetoResult {
+ public:
+  /** Constructor */
+  VisiblesVetoResult();
+
+  /** Destructor */
+  virtual ~VisiblesVetoResult();
+
+  void setVariables(int nLayersHit, double xStd, double yStd, double zStd,
+                    double xMean, double yMean, double rMean, int isoHits,
+                    double isoEnergy, int nReadoutHits, double summedDet,
+                    double rMeanFromPhotonProj);
+
+  void Clear();
+
+  void Print() const;
+
+  bool passesVeto() const { return passesVeto_; }
```

Comment:
```suggestion
  bool passesVeto() const { return passes_veto_; }
```
etc, same comment about naming conventions

Reply:
This is a comment for the whole PR, but I wont point it out for all the cases, please just apply the rules everywhere